react-js/react.md

```diff
@@ -47,53 +50,65 @@ You will spend the day working through these materials.
 
 ### Topics to learn & Facilitate
 
-- What is React?
-  - React is a library for creating User Interfaces.
-  - This means it is used for creating what some refer to as the "view layer" of an application.
-- JSX adds to our JavaScript to allow for HTML-like markup in JavaScript.
-- Show Hello World example, plus expanded example.
-- Example for appending to a list in jQuery vs. the same in React.
+- React lets you generate HTML using JavaScript
+- The HTML can change over time, unlike regular HTML
+- No jQuery needed
+
+### Examples
+#### CodePen - Hello World component
+In this CodePen, a basic Hello World component is rendered.<br> 
+Try modifying line 5 of the JS file to see the changes made to the file rendered.
+<br>
+[Link](https://codepen.io/jamesjose03/pen/ExVjYBM)
```

Comment:
Sorry for not being clear: it's not the specific language of "Click here" that makes it an anti-pattern, just that it provides no context as to where it will take you. Please change the label to CodePen of Step 1, etc. Again, please change this for all instances. I'm not going to mark them all so text search yourself before revising to make sure you caught them all.

In the future, if you're not sure you understand a revision suggestion, it would be better to ask for clarification.

Reply:
Changed it 👍